crates/symbolicator/src/config.rs

```diff
@@ -243,9 +243,15 @@ pub struct Config {
     /// Number of subprocesses in the internal processing pool.
     pub processing_pool_size: usize,
 
-    /// The timeout for downloads.
+    /// The maximum timeout for downloads.
```

Comment:
I'd describe the behaviour a little more explicit as well.  That means anyone refactoring any of this in the future knows exactly what behaviour was important to uphold.  Something like:
```suggestion
    /// The maximum timeout for downloads.
    ///
    /// This is the upper limit the download service will take for downloading from a single
    /// source, regardless of how many retries are involved.
```